protos/tune/tune.proto

```diff
@@ -5,23 +5,26 @@ package mavsdk.rpc.geofence;
 option java_package = "io.mavsdk.tune";
 option java_outer_classname = "TuneProto";
 
+// Sends a tune to be played by the vehicle
 service TuneService {
-  /*
-   * Send a tune to be played by the system.
-   */
+  // Send a tune to be played by the system.
   rpc PlayTune(PlayTuneRequest) returns(PlayTuneResponse) {}
 }
 
+// represents an element of the tune
 enum SongElement {
+  // playing styles
```

Comment:
Hehe I'm sorry about that, but the parser won't work with it :sweat_smile:. It has to be after each element, e.g.:

```
STYLE_LEGATO = 0; // After this element, start playing legato
STYLE_NORMAL = 1; // After this element, start playing normal
STYLE_STACCATO = 2; // After this element, start playing staccato
```

I understand it's a bit annoying, but that it then used to generate the API documentation directly in the code (currently only in Python) :+1:.